examples/hacker_news_assets/hacker_news_assets/repo.py

```diff
@@ -28,3 +32,8 @@ def hacker_news_assets_staging():
         make_hn_tables_updated_sensor(activity_stats_staging_job),
         make_hn_tables_updated_sensor(story_recommender_staging_job),
     ]
+
+
+@repository
+def hacker_news_assets_local():
+    return [local_assets, download_local_job]
```

Comment:
why does this one only include the download job? Why are the activity_stats, .... and others specified for PROD and STAGING missing in local?

Reply:
Even when trying to add them:

dagster.core.errors.DagsterInvalidDefinitionError: Input asset '["activity_daily_stats"]' for asset 'activity_forecast' is not produced by any of the provided asset ops and is not one of the provided sources

Reply:
To your second question, that error will stop showing up when this gets merged: https://github.com/dagster-io/dagster/pull/6967.

That change is already on the duckdb branch. If you check out the `hn-duckdb` branch and `pip install -e python_modules/dagster`, the error should go away.

Reply:
> why does this one only include the download job? Why are the activity_stats, .... and others specified for PROD and STAGING missing in local?

No good reason - just hadn't gotten around to it yet. I intend to add those as well.

Reply:
When do you plan to merge the #6967?

Reply:
your suggestion of: `pip install -e python_modules/dagster` only partially works:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
dagstermill 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-spark 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-slack 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-shell 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-pyspark 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-postgres 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-pandera 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-pandas 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-graphql 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-dbt 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagster-airbyte 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
dagit 0.14.3 requires dagster==0.14.3, but you have dagster dev which is incompatible.
```